core/controller/src/main/scala/whisk/core/controller/ApiUtils.scala

```diff
@@ -166,16 +166,16 @@ trait ReadOps extends Directives {
                                                                                         ma: Manifest[A]) = {
     onComplete(factory.get(datastore, docid)) {
       case Success(entity) =>
-        logging.info(this, s"[GET] entity success")
+        logging.debug(this, s"[GET] entity success")
         postProcess map { _(entity) } getOrElse complete(OK, entity)
       case Failure(t: NoDocumentException) =>
-        logging.info(this, s"[GET] entity does not exist")
+        logging.warn(this, s"[GET] entity does not exist")
```

Comment:
the use of `warn` here will lead to a lot of spurious warnings that can be caused by bad client requests. I suggest using `debug` instead. This will apply to several of the error cases below.